### DISCUSSION TITLE
Implementation autoupdate when new version release.

### DISCUSSION THREAD

**Discussion by BElluu** [godotengine/godot-proposals#8553] [Editor]
### Describe the project you are working on

-

### Describe the problem or limitation you are having in your project

-

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add autoupdate mechanism.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

-

### Is there a reason why this should be core and not an add-on in the asset library?

-

**Comment by AThousandShips** [comment]
The questions on the form are *not* optional, you have to fill in each one, as there's no actual ideas for how to do this I will convert this to a discussion, but please respect the process in the future

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1132.
